sdk/cosmos/cosmosclient/src/macros.rs

```diff
@@ -0,0 +1,21 @@
+macro_rules! c_str {
+    ($s:expr) => { const {
```

Comment:
`CStr` can't do this already?

Reply:
Not that I've found, no. There are CStr literals but we want to read in an environment variable.